features/step_definitions/login_steps.rb

```diff
@@ -1,14 +1,23 @@
 Given(/^the login page$/) do
   #put your code here
-  pending
+  visit '/'
 end
 
-When(/^I log in with proper credentials$/) do
-  #put your code here
-  pending
+When(/^I log ?in with user '([^']+)' and password '([^']+)'$/) do |user,pass|
+  @username = user
+  within('form') do
+    fill_in 'username', :with => user
+    fill_in 'password', :with => pass
```

Comment:
I didn't know that was the difference between those styles. Learn something new.